packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -0,0 +1,753 @@
+// Internal dependencies.
+import {WebexPlugin} from '@webex/webex-core';
+import {safeSetTimeout} from '@webex/common-timers';
+
+import FeaturesModel from './features-model';
+
+/**
+ * Devices plugin.
+ *
+ * @description
+ * This plugin is used as a prototyped replacement plugin for the previous
+ * device plugin `internal-plugin-wdm`. After this plugin is fully implemented,
+ * other plugins will likely need to be modified to consume this one over
+ * `internal-plugin-wdm` [ **device** ].
+ */
+const Devices = WebexPlugin.extend({
+
+  // Ampersand property members.
+
+  namespace: 'Devices',
+
+  children: {
+    /**
+     * The class object that contains all of the feature collections.
+     *
+     * @type {FeaturesModel}
+     */
+    features: FeaturesModel
+  },
+
+  props: {
+    /**
+     * This property determines whether or not giphy support is enabled.
+     *
+     * @type {'ALLOW' | 'BLOCK'}
+     */
+    clientMessagingGiphy: 'string',
+
+    /**
+     * This property appears to be unused, but should store the company name.
```

Comment:
What do you mean `This property appears to be unused`?

Reply:
When implementing, I didn't see any use case in which those properties were being used, even in the retrieved payload from WDM. I'll remove the `unused` assertion and leave it as `This property should store...` as the tone makes more sense this way.

Reply:
They are being used in the web client.